flask_wiki/api.py

```diff
@@ -187,6 +190,12 @@ def render(self):
         processor = Processor(self.content)
         self._html, self.body, self._meta, self.toc = processor.process()
 
+        # Get creation and update times from file
+        self.modified_time = time.localtime(self.stat[stat.ST_MTIME])
```

Comment:
I would use `os.path` instead of `os.stat` as we need only few informations:

```python
os.path.getmtime(path)
#and
os.path.getctime(path)
```

Reply:
Instead of having 4 variables I would have only a creation_datetime and modification_datetime. This can be formatted to be well displayed.